packages/flutter_tools/test/general.shard/ios/mac_test.dart

```diff
@@ -113,7 +123,7 @@ void main() {
       Artifacts: () => mockArtifacts,
     });
 
-    testUsingContext('getInfoForDevice throws IOSDeviceNotFoundError when user has not yet trusted the host', () async {
+    testUsingContext('getInfoForDevice throws IOSDeviceLockdownError with LockdownReturnCode.pairingDialogResponsePending when user has not yet trusted the host', () async {
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters

It looks like you could break some of these up using multi-line strings or splitting parameter lists on ','.